docs/api/CheerioCrawler.md

```diff
@@ -47,7 +47,7 @@ await requestList.initialize();
 // Crawl the URLs
 const crawler = new Apify.CheerioCrawler({
     requestList,
-    handlePageFunction: async ({ request, response, html, $ }) => {
+    handlePageFunction: async ({ request, response, body, html, $ }) => {
```

Comment:
I think that we should deprecate the `html` in favor of `body`. No point in having both.